src/content/docs/tutorial-kubernetes-learn/tutorial-k8s-layers.mdx

```diff
@@ -0,0 +1,185 @@
+
+Use the time selector in the top left of the page to see your data across time ranges to verify any troubling data isn't just random or to triage across a longer timeframe. 
+    </Step>
+</Steps>
+
+## Understand and monitor the orchestrated layer
+
+Continuing with our bakery analogy, the orchestrated layer is the baking stations within the bakery. Each station houses a set of ovens that work together to fulfill specific tasks. Each station might be designated to have three ovens running at all times. If an oven fails, the station will fire up a new oven to keep up with expectations.
+
+In a Kubernetes system, deployments have various specifications. For example, a deplyoment might be specified to always run four instances of an application in individual containers. The deployment will spin up pods containing that application until it meets that quota. If a pod were to fail, it would spin up a new one to continue to meet the specified  .
+
+
+The following steps guide you through a general strategy to monitor your deployments:
+
+<Steps>
+    <Step>
+### Navigate to the overview dashboard
+
+Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Kubernetes > Overview Dashboard**.         
+    </Step>
+    <Step>
+### Triage your deployments
+
+Take a look at the unhealthy deployment tile and the deployments tile. Compare the two to measure the percentage of unhealthy deployments in your cluster. Unhealthy deployments are deployments with missing or unavailable pods. This usually means that the deployment was unable to spin up those pods. Let's take a close look.
+
+<img
+    title="tutorialOverviewdashboard"
+    alt="The main overview dashboard for the kubernetes capability"
+    src={tutorialOverviewDeployments}
+/>
+
+    </Step>
+    <Step>
+        
+### Identify pending and failed pods
+
+Scroll down a bit and find the **Pending and Failed Pods** table. This table will show you all the pods that have failed or are stuck pending for whatever reason. It's normal that pods fail at to a certain degree depending on the baseline health of your system. What you're looking for is pods that repeatedly fail. Scroll down the chart as needed. Once you've identified which pods have are failing regularly, troubleshoot the deployment configs for those pods.
+
+<img
+    title="tutorialOverviewFailed"
+    alt="The main overview dashboard for the kubernetes capability focusing on the failed pods section"
+    src={tutorialOverviewFailed}
+/>
+
+    </Step>
+</Steps>
+
+### Understand and monitor the service and application layer
+
+In your bakery, a baker runs each individual oven. These bakers are the heart of the system, providing the actual value — the bread!. 
+
+In your kubernetes system, each pod contains services and applications that provide the actual functionality that your kubernetes system supports. This could be computation, a web app, or anything inbetween.
```

Comment:
Can eliminate the passive in the second sentence by changing it to "system supports, such as computation..."